sunpycube/cube/NDCube.py

```diff
@@ -165,7 +167,18 @@ def world_to_pixel(self, quantity_axis_list, origin=0):
         return result[::-1]
 
     def to_sunpy(self):
-        raise NotImplementedError("to_sunpy() is not Implemented.")
+        wcs_axes = list(self.wcs.wcs.ctype)
+        missing_axis = self.missing_axis
+        index_not_one = []
+        if 'TIME' in wcs_axes and len(self.dimensions.shape) is 1:
+            result = self.pixel_to_world([u.Quantity(self.data, unit=u.pix)])
```

Comment:
@DanRyanIrish it makes very little sense to me. I don't understand what you are trying to do.